### PR TITLE
*: update deps

### PR DESCRIPTION
**Pull request by BusyJay**
make github dependant bot happy.

### CODE REVIEW

**Review by skyzh — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -48,7 +48,9 @@ jobs:
       - name: Running Tests With Code Coverage 🚀
         run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
       - name: Upload Coverage To Codecov ⬆️
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```
Comment (tabokie) [inline .github/workflows/test.yml:53]: Somehow this is overwriting raft-engine's coverage: https://app.codecov.io/gh/tikv/raft-engine

Comment (skyzh) [reply .github/workflows/test.yml:53]: I don't have admin access to the repo so someone else needs to generate a new token and fix that :)

Comment (BusyJay) [reply .github/workflows/test.yml:53]: The token is copied from the repo page, strange it will overwriting another repo 🤔

Comment (BusyJay) [reply .github/workflows/test.yml:53]: I setup tokens for both agatedb and raft-engine, and coverage works fine for agatedb now. Not sure if it's the cause, it's also possible that a wrong key is used in the first place.

**Pull request merged**